libraries/ESP8266WiFi/src/ESP8266WiFiGeneric.cpp

```diff
@@ -718,6 +718,48 @@ void wifi_dns_found_callback(const char *name, const ip_addr_t *ipaddr, void *ca
     esp_schedule(); // break delay in hostByName
 }
 
+void wifi_dns_found_callback_async(const char* name, const ip_addr_t* ipaddr, void* callback_arg);
```

Comment:
`static` as it should not exist out of this file

Reply:
Done in df3075cd47a18f96de540cf2dc5a994fbcbf5a84